package.json

```diff
@@ -16,42 +16,26 @@
     "tdd": "ava --watch"
   },
   "dependencies": {
-    "asar": "^0.11.0",
+    "asar": "^1.0.0",
     "bluebird": "^3.3.4",
     "debug": "^2.2.0",
-    "fs-extra": "^2.1.2",
+    "fs-extra": "^7.0.1",
     "lodash.template": "^4.2.2",
+    "pify": "^4.0.1",
     "temp": "^0.8.3"
   },
   "devDependencies": {
     "ava": "^0.13.0",
     "babel-cli": "^6.6.5",
     "babel-eslint": "^6.0.2",
     "babel-plugin-transform-async-to-module-method": "^6.7.0",
-    "babel-plugin-transform-runtime": "^6.6.0",
-    "babel-preset-es2015-node4": "^2.0.3",
-    "babel-preset-stage-0": "^6.5.0",
+    "babel-preset-node6": "^11.0.0",
```

Comment:
The modern convention is to use `babel-preset-env`, rather than any other `babel-preset-*` for compilation targets.